templates/taxi/car_detail.html

```diff
@@ -12,6 +12,16 @@ <h1>
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+  {% if user not in car.drivers.all %}
```

Comment:
Ensure that the conditional logic `{% if user not in car.drivers.all %}` correctly checks if the user is not already a driver of the car. This logic should be verified to ensure it functions as expected.